hooks/pre/30-upgrade.rb

```diff
@@ -80,6 +79,30 @@ def fix_katello_settings_file
   end
 end
 
+def mongo_mmapv1_check
+  custom_hiera = '/etc/foreman-installer/custom-hiera.yaml'
+  mongodb_dir = '/var/lib/mongodb/'
+  # check and see if we have a pulp_database already, we have a wiredTiger file, and if the Hiera file has already been modifed.
+  if File.file?("#{mongodb_dir}/pulp_database.0") && File.file?("#{mongodb_dir}/WiredTiger.wt")
```

Comment:
So if the pulp database exists, and a wiredTiger file does not exist, we skip this block? I think this block was needed to ensure we kept `mmapv1`?